### PR TITLE
Added API Changes

### PR DESCRIPTION
**Pull request by abhishek246006**

### CODE REVIEW

File: util/env.go [hunk 1]
```diff
@@ -1,10 +1,42 @@
...
 func GetConfigValue(key string) string {
 	//TODO: include AWS Parameter Store
-	return os.Getenv(key)
+	value := os.Getenv(key)
+	if value != "" {
+		return value
+	}
+	prefix := os.Getenv("SSM_PS_NP")
+	region := os.Getenv("SSM_PS_RG")
+	sess, err := session.NewSessionWithOptions(session.Options{
```
Comment (happay) [inline util/env.go:20]: @abhishek246 Can you make the **sess** and **ssmsvc** objects as local variable on this file?

**Review by happay — COMMENTED** [review]
@abhishek246006  Could you make the **sess** and **ssmsvc** objects as local variable on the util/env.go file?

**Review by happay — COMMENTED** [review]
i think we should create an **aws** folder inside the connector and different go files for sqs, ses, s3, etc inside the **aws** folder.

File: connector/aws/ses.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"strings"
+)
+
+// ============ Constants =============
+
+const (
+	CharSet        = "UTF-8" // The character encoding for the email.
+	NoReplyEmailId = "no-reply@happay.in"
```
Comment (happay) [inline connector/aws/ses.go]: What is the use of `NoReplyEmailId ` and `MatrixGroupEmail `?

**Comment by abhishek246006** [comment]
Please review and merge

**Review by happay — COMMENTED** [review]
Ideally, all the AWS functionalities should be inside their own package.
aws --> cred.go - This goes in aws package.
aws --> s3 --> s3.go - This goes in s3 package.
aws --> ses --> ses.go - This goes in ses package.
and so on.

**Review by happay — CHANGES_REQUESTED** [review]
@abhishek246006  I have put some comment. Please go through it and make the necessary changes.

**Comment by abhishek246006** [comment]
@happay I have made the changes requested

**Pull request merged**